### PR TITLE
KAFKA-5481: ListOffsetResponse isn't logged in the right way with trace level enabled

### PR DESCRIPTION
**Pull request by ppatierno**
Added toString() method to ListOffsetResponse for logging

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-jdk8-scala2.12/5489/
Test PASSed (JDK 8 and Scala 2.12).

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-jdk7-scala2.11/5504/
Test PASSed (JDK 7 and Scala 2.11).

File: clients/src/main/java/org/apache/kafka/common/requests/ListOffsetResponse.java [hunk 1]
```diff
@@ -191,4 +191,15 @@ protected Struct toStruct(short version) {
...
     }
+
+    @Override
+    public String toString() {
+        StringBuilder bld = new StringBuilder();
+        bld.append("(type=ListOffsetResponse")
+            .append(", throttleTimeMs=").append(throttleTimeMs)
+            .append(", responseData=").append(responseData);
+
+        bld.append(")");
```
Comment (ewencp) [inline clients/src/main/java/org/apache/kafka/common/requests/ListOffsetResponse.java]: *Really* minor nit, but can we chain this `append` with all the others like we do in other implementations of request/response `toString()` methods?

**Review by ewencp — COMMENTED** [review]
One extremely minor style nit, then LGTM

File: clients/src/main/java/org/apache/kafka/common/requests/ListOffsetResponse.java (hunk 1 above)
Comment (ppatierno) [reply clients/src/main/java/org/apache/kafka/common/requests/ListOffsetResponse.java]: @ewencp just done, totally agree ... I don't know why it was there ;)

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-jdk7-scala2.11/6244/
Test PASSed (JDK 7 and Scala 2.11).

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-jdk8-scala2.12/6228/
Test PASSed (JDK 8 and Scala 2.12).

**Review by ewencp — COMMENTED** [review]
Awesome, thanks for the patch! Merging to trunk.

**Pull request closed without merging**